### ISSUE TITLE
maskfeat mvit-s fine-tuning on k400 config

### ISSUE THREAD

**Issue by cir7** [facebookresearch/SlowFast#627]
Hi, I am trying to reproduce mvit-s fine-tuning on k400, and find that the config file (configs/masked_ssl/k400_MVITv2_S_16x4_FT.yaml) is different from the config in the paper in the `SOLVER` part, in config file, we warmup for 20 epochs, and max epoch is 100, but in paper, warmup for 5 epochs, and training for 200 epochs. which config should i refer to? 
Another question is that MViT-L is fine-tuning on larger saptial sizes  of 312 and 352,  are you using a higer resolution (larger than short-side 256) version k400 dataset?
